codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/generators/protocol/MakeOperationGenerator.kt

```diff
@@ -38,7 +39,9 @@ open class MakeOperationGenerator(
     private val protocol: Protocol,
     private val bodyGenerator: ProtocolPayloadGenerator,
     private val functionName: String = "make_operation",
-    private val public: Boolean = true
+    private val public: Boolean = true,
+    /** Whether or not to include default values for content-length and content-type */
+    private val includeDefaultPayloadHeaders: Boolean = true,
```

Comment:
consider removing the default values here—since this function is only invoked in 2 (?) places, explicit is probably clearer on both sides